test/choosetests.jl

```diff
@@ -82,59 +82,27 @@ function choosetests(choices = [])
         tests = testnames
     end
 
-
-    unicodetests = ["unicode/utf8"]
-    if "unicode" in skip_tests
-        filter!(x -> (x != "unicode" && !(x in unicodetests)), tests)
-    elseif "unicode" in tests
-        # specifically selected case
-        filter!(x -> x != "unicode", tests)
-        prepend!(tests, unicodetests)
-    end
-
-    stringtests = ["strings/basic", "strings/search", "strings/util",
-                   "strings/io", "strings/types"]
-    if "strings" in skip_tests
-        filter!(x -> (x != "strings" && !(x in stringtests)), tests)
-    elseif "strings" in tests
-        # specifically selected case
-        filter!(x -> x != "strings", tests)
-        prepend!(tests, stringtests)
+    function patchlib!(tests, name, files=[name])
```

Comment:
Calling this `patchlib!` is a little weird. That makes me think of patchelf and other such library patching utils.

Reply:
1. probably not the best name one can choose. first choice was the last one o:

2. functional prg @ code leads to delta-oriented programming @ data. 
    diff, patch etc can spread and disseminate. that's great!

3. no problem at all to change this name. 
    i will try to get inspiration for that. hum, hum.

Reply:
The new name `filtertests!` is indeed more descriptive!